.github/workflows/npm-release-publish.yml

```
@@ -71,6 +71,25 @@ jobs:
           git tag ${{ steps.version.outputs.version }}
           git push --tags
 
+      # Get datetime to name and tag the releases with.
+      - name: Get Release names
+        id: releasenames
+        run: |
+          echo "::set-output name=tagname::$(TZ=":America/Vancouver" date +'%Y-%m-%d-%H%M')"
+          echo "::set-output name=releasename::$(TZ=":America/Vancouver" date +'%Y-%m-%d-%H%M')"
+
+      # Upload releases
+      - name: Upload Public Preview Release
+        uses: softprops/action-gh-release@v1
+        with:
+          name: PublicPreview ${{ steps.releasenames.outputs.releasename }}
+          tag_name: Release/${{ steps.releasenames.outputs.tagname }}
```

For the name should we just use the same as the npm version?
```suggestion
          tag_name: ${{ steps.version.outputs.version }}
```